javascript/modules/client/test/integration/client-addressList/client-addressList.test.ts

```diff
@@ -0,0 +1,60 @@
+// @ts-ignore
```

Comment:
Isn't the name of the file repetitive? Not a big deal though

We could simply call it `client` or even `index`